File: Submissions/#0332_div2/A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  int d1, d2, d3;
  cin >> d1 >> d2 >> d3;

  int answer = d1 + d2 + d3;

  answer = min(answer, d1 + d1 + d2 + d2);

  answer = min(answer, d1 + d1 + d3 + d3);

  answer = min(answer, d2 + d2 + d3 + d3);

  cout << answer << endl;

}

File: Submissions/#0332_div2/A/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);
    int d1 = sc.nextInt();
    int d2 = sc.nextInt();
    int d3 = sc.nextInt();
    sc.close();

    int answer = d1 + d2 + d3;

    answer = Math.min(answer, d1 + d1 + d2 + d2);

    answer = Math.min(answer, d1 + d1 + d3 + d3);

    answer = Math.min(answer, d2 + d2 + d3 + d3);

    System.out.println(answer);

  }

}